geniza/footnotes/migrations/0005_add_permissions_source_sourcetype_footnotes.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 3.1.7 on 2021-04-01 15:11
+
+from django.contrib.auth.management import create_permissions
+from django.db import migrations
+
+# Models:
+#   sourcetype, source, footnote
+# Permissions:
+#   view_sourcetype
+#   add_sourcetype
+#   change_sourcetype
+#   delete_sourcetype
+#   view_source
+#   add_source
+#   change_source
+#   delete_source
+#   view_footnote
+#   add_footnote
+#   change_footnote
+#   delete_footnote
+
+CONTENT_EDITOR = 'Content Editor'
+# new permissions for content editor
+content_editor_perms = [
+    'view_sourcetype', 'add_sourcetype', 'change_sourcetype',
+    'view_source', 'add_source', 'change_source',
+    'view_footnote', 'add_footnote', 'change_footnote',
+]
+
+
+CONTENT_ADMIN = 'Content Admin'
+# additional new permissions for content admin
+content_admin_perms = [
+    'delete_sourcetype',
+    'delete_source',
+    'delete_footnote',
+]
+
+
+def create_content_editor_groups(apps, schema_editor):
```

Comment:
Now that I see it here, it makes a lot more sense for the migration for footnote-specific permissions to be in the footnotes app than in the common one. 

Is it worth revising the documents-fragments permissions migration to follow that convention?

Also, I wonder if we want a migration utility method for adding permissions...